File: Veterinario/ConsultorioVeterinario/ConsultorioVeterinario/Logica/Medicamento.cs
namespace ConsultorioVeterinario.Logica
{
    public class Medicamento
{
    public string Codigo { get; set; }
    public string Nombre { get; set; }
    public int Stock { get; set; }

    public Medicamento(string codigo, string nombre, int stock)
    {
        Codigo = codigo;
        Nombre = nombre;
        Stock = stock;
    }
}
}

File: Veterinario/VeterinarioBlueJ/Medicamento.java
public class Medicamento {
    private String codigo;
    private String nombre;
    private int stock;

    public Medicamento(String codigo, String nombre, int stock) {
        this.codigo = codigo;
        this.nombre = nombre;
        this.stock = stock;
    }

    public String getCodigo() { return codigo; }
    public String getNombre() { return nombre; }
    public int getStock() { return stock; }
    public void setStock(int stock) { this.stock = stock; }
}
